doc/before-after-reboot-checks.md

```diff
@@ -0,0 +1,46 @@
+# Before and After Reboot Checks
+
+CLUO can require custom node annotations before a node is allowed to reboot or
+before a node is allowed to become schedulable after a reboot. 
+
+## Configuring the `update-operator`
+
+When depolying the `update-operator`, you can provide a comma-separated list of
```

Comment:
Perhaps:

Configure `update-operator` with comma-separated lists of `--before-reboot-annotations` and `--after-reboot-annotations` that should be required.